packages/libs/eda/src/lib/core/components/computations/Types.ts

```diff
@@ -51,7 +51,7 @@ export interface ComputationPlugin {
   }>;
   visualizationPlugins: Partial<Record<string, VisualizationPlugin<any>>>;
   createDefaultConfiguration: (
-    rootEntity: StudyEntity
+    rootEntity?: StudyEntity
```

Comment:
I didn't find `rootEntity` to be used in any app's default config. It's likely we can just remove, but testing the waters by making it optional first